bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/EdProgram.java

```diff
@@ -244,7 +244,9 @@ private static void setTimeLabel(final long time, final JLabel timeLabel) {
         timeLabel.setText(timeStr);
     }
     private static void setTimeLabel(final double time, final JLabel timeLabel) {
-        final String timeStr = time == 0d ? ZERO_TIME_STRING : new HMS(time).toString();
+        final String timeStr = time == 0d ?
+                ZERO_TIME_STRING :
+                new HMS(time).toString(true, false);
```

Comment:
I guess not today, but ideally we'd scrap this use of `HMS` in favor of the `java.time` API.